docs/intro.rst

```diff
@@ -116,13 +118,38 @@ If the same property is set with multiple methods,
 the order of precedence is command-line properties, external properties file and
 classpath properties file. You can find a list of settings in :ref:`configuration section<camus_config>` .
 
+The recommended deployment method is to use the ``camus-run`` script to initiate the MapReduce job:
+
+.. sourcecode:: bash
+
+   $ bin/camus-run -D <property=value> \
+      -P <path to external properties file> \
+      -p <path to properties file from classpath>
+
+If you need more control you can run the job yourself, but will have to configure some parameters
+and environment variables yourself (or reuse the ``bin/camus-config`` script to generate the
+configs without running the job). If you install Camus via zip/tgz archive,
+you can find Camus's jar files under ``share/java/camus/``. If you install Camus via rpm or deb,
+the Camus's jar files under ``/usr/share/java/camus/``.
+
 .. sourcecode:: bash
 
-   $ hadoop jar confluent-camus-1.0-shaded.jar com.linkedin.camus.etl.kafka.CamusJob
-      -D <property=value>
-      -P <path to external properties file>
+   # Assuming:
+   # 1. hadoop is on your PATH
+   # 2. HADOOP_CLASSPATH includes all the Camus jars
+   # 3. HADOOP_USER_CLASSPATH_FIRST=true
+   # 4. CAMUS_LIBJARS contains the comma-separated list of all Camus jars
+
+   $ hadoop jar confluent-camus-$VERSION.jar com.linkedin.camus.etl.kafka.CamusJob \
+      -libjars $CAMUS_LIBJARS -D mapreduce.job.user.classpath.first=true \
+      -D <property=value> \
+      -P <path to external properties file> \
       -p <path to properties file from classpath>
 
+
+For some Hadoop distributions, you may be able to remove some of these settings to simplify the
+command.
```

Comment:
libjars always should be required, right? How else would all the jars be available for the tasks, which run on other nodes? I think the only case this worked was when it was mistakenly running locally since then we only needed the jars to be on the classpath?

I added a note about `HADOOP_USER_CLASSPATH_FIRST` and  `mapreduce.job.user.classpath.first`.